code/modules/mob/living/living.dm

```diff
@@ -749,19 +749,25 @@
 // Override if a certain mob should be behave differently when placing items (can't, for example)
 /mob/living/stripPanelEquip(obj/item/what, mob/who, where, var/silent = 0)
 	what = get_active_hand()
-	if(what && (what.flags & NODROP))
-		to_chat(src, "<span class='warning'>You can't put \the [what.name] on [who], it's stuck to your hand!</span>")
-		return
 	if(what)
+		if(what.flags & NODROP)
+			to_chat(src, "<span class='warning'>You can't put \the [what.name] on [who], it's stuck to your hand!</span>")
+			return
 		if(!what.mob_can_equip(who, where, 1))
+			if((where == slot_l_hand || where == slot_r_hand) && isliving(who))
+				var/mob/living/L = who
+				if(L.resting)
+					to_chat(src, "<span class='warning'>[who.name] can't hold \the [what.name] while resting.</span>")
+					return
```

Comment:
Why have these checks at all?

Reply:
If somebody is lying down without these checks it'll say that you can't put that item there. Bit of an odd warning I found.

Reply:
having redundant code to check for the same thing that's already checked is icky, just have it be default behavior.

Reply:
Will do